wiki/cardinal-components-api/dev-install.md

```diff
@@ -4,10 +4,69 @@ breadcrumb: Dev Installation
 layout: cca_wiki
 ---
 
+<script src="/scripts/modrinthVersionCheck.js"></script>
```

Comment:
Not familiar enough with template-based files, but how challenging would it be to add this to a layout template or similar with an option to enable or disable the inclusion of the script in the head tag?